src/Patients/CreatePatient.jsx

```diff
@@ -7,12 +7,13 @@ import { useForm } from 'react-hook-form';
 import './CreatePatient.scss';
 const axios = require('axios');
 
-const CreatePatient = () => {
+const CreatePatient = (props) => {
```

Comment:
Just deconstruct props `{props}` and then use as and possibly a different name from `props`.

Reply:
const CreatePatient = ({ handleClose }) => {...